src/cmd_line/commands/file.ts

```diff
@@ -85,6 +86,19 @@ export class FileCommand extends node.CommandBase {
       let folder = vscode.Uri.file(newFilePath);
       await vscode.commands.executeCommand("vscode.open", folder,
         this._arguments.position === FilePosition.NewWindow ? this.getViewColumnToRight() : this.getActiveViewColumn());
+
+      await vscode.commands.executeCommand("cursorMove", {
```

Comment:
No, go to line. The file action looks good to me, then jumping to the right line needs to be cleaned up